### DISCUSSION TITLE
Had some wired issue that I can't reproduce so I record it here

### DISCUSSION THREAD

**Discussion by 117649** [onemen/TabMixPlus#284] [General]
@onemen 
I set middle-click on tab to open last closed tab.
And I encountered this problem that when middle-click on a tab it also close the tab that been clicked beside open the last closed one.
The problem is gone by itself after several tries including close more tabs then reopen them.
Here is some alert in the log I'm feeling may be related. 
![image](https://github.com/onemen/TabMixPlus/assets/8281345/a4c41760-2de0-48de-8b36-dce3f3bb4f60)
![image](https://github.com/onemen/TabMixPlus/assets/8281345/2c59765c-3c77-406f-91d5-5db006ddf645)

**Comment by onemen** [comment]
Post your Tab Mix Plus preferrnces

**Reply by 117649** [reply]
[TMPpref.txt](https://github.com/onemen/TabMixPlus/files/14704909/TMPpref.txt)

**Comment by onemen** [comment]
I can't reproduce this issue.

**Reply by 117649** [reply]
I can't neither it only occur for a limit period and disappeared by itself.
But I have to say FF is giving more wired behavior after prolonged period of using recently.

**Comment by onemen** [comment]
try to reproduce it with more than one window open

**Reply by 117649** [reply]
I don't really how.
All I only know is this could happen if I let FF running for hours even days. And I don't recall ever seen this before 123.

**Comment by onemen** [comment]
Just pushed https://github.com/onemen/TabMixPlus/commit/f22faaa5661d6498f55eee44a4bff5c08e2682b9 to fix this issue.
Download and install latest [Development Build](https://github.com/onemen/TabMixPlus/releases/tag/dev-build) to test it.

Firefox has 2 new preferences:
  browser.sessionstore.closedTabsFromAllWindows - [Bug 1819675](https://bugzilla.mozilla.org/show_bug.cgi?id=1819675)
  browser.sessionstore.closedTabsFromClosedWindows- [Bug 1833522](https://bugzilla.mozilla.org/show_bug.cgi?id=1833522)

![image](https://github.com/onemen/TabMixPlus/assets/3650909/49ba8607-7481-4a85-95c2-3ad3705fa040)

**Reply by 117649** [reply]
Actually I haven't see this problem for a second time.
